gr-uhd/apps/uhd_app.py

```diff
@@ -364,12 +392,11 @@ def cslist(string):
                            help="Sample rate")
         group.add_argument("-g", "--gain", type=eng_arg.eng_float, default=None,
                            help="Gain (default is midpoint)")
-        group.add_argument("--gain-type", choices=('db', 'normalized'), default='db',
-                           help="Gain Type (applies to -g)")
-        group.add_argument("-p", "--power-ref", type=eng_arg.eng_float, default=None,
-                           help="Reference power level (in dBm). "
+        group.add_argument("-p", "--power", type=eng_arg.eng_float, default=None,
+                           help="(Reference) power level (in dBm). "
                                 "Not supported by all devices (see UHD manual). "
-                                "Will fail if not supported. Precludes --gain. ")
+                                "Will fail if not supported. Precludes --gain. "
+                                "Behaviour may differ between applications.")
```

Comment:
Since `--power` is related to `--amplitude`, I think this relationship should be noted ... here and/or with that argument.

Reply:
`uhd_app.py` shouldn't have `--amplitude` anyway, that's specific to siggen. I'll move it out.

Reply:
To clarify: Changing `--amplitude` does not change the output power (UHD will shift gains around accordingly). I'll add a comment.